deployment/v3/external/landing-page/install.sh

```diff
@@ -12,6 +12,9 @@ CHART_VERSION=12.0.2
 echo Create $NS namespace
 kubectl create ns $NS
 
+read -p "Select ConfigMap landing page (index or collab-index) [default: index]: " CONFIG_MAP_OPTION
```

Comment:
Select ConfigMap for landing page (index or collab-index) [default: index]:

Reply:
updated